plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/command/AssociationCreateCommand.java

```diff
@@ -167,19 +181,25 @@ protected ERDAssociation createAssociation(ERDElement<?> sourceEntity, ERDElemen
             DBVEntity vEntity = DBVUtils.getVirtualEntity(srcEntityObject, true);
             assert vEntity != null;
 
-            DBVEntityForeignKey vfk = EditForeignKeyPage.createVirtualForeignKey(
+            Supplier<DBVEntityForeignKey> virtualFkSupplier = EditForeignKeyPage.makeVirtualForeignKeySupplier(
                 vEntity,
                 targetEntityObject,
                 new EditForeignKeyPage.FKType[] {
                     EditForeignKeyPage.FK_TYPE_LOGICAL
                 },
                 srcAttrs,
                 refAttrs);
+            DBVEntityForeignKey vfk = virtualFkSupplier.get();
             if (vfk == null) {
+                this.virtualFk = null;
+                this.virtualFkSupplier = null;
                 return null;
+            } else {
+                vEntity.persistConfiguration();
```

Comment:
Configuration is already persisting in `ERDEditorEmbedded::doSave`. Not sure if it should be done here too.